libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/dragDropAndMoveWidget.spec.ts

```diff
@@ -0,0 +1,80 @@
+// (C) 2023 GoodData Corporation
+
+import * as Navigation from "../../tools/navigation";
+import { Widget } from "../../tools/widget";
+import { EditMode } from "../../tools/editMode";
+import { Dashboard } from "../../tools/dashboards";
+import { InsightsCatalog } from "../../tools/insightsCatalog";
+import { DashboardMenu } from "../../tools/dashboardMenu";
+import { DashboardHeader } from "../../tools/dashboardHeader";
+
+const dashboard = new Dashboard();
+
+Cypress.Cookies.defaults({
+    preserve: ["GDCAuthTT", "GDCAuthSTT", "_csrfToken"],
+});
+
+Cypress.on("uncaught:exception", (error) => {
+    console.error("Uncaught exception cause", error);
+    return false;
+});
+
+Cypress.Cookies.debug(true);
+
+describe("Insight on dashboard", { tags: ["pre-merge_isolated_tiger"] }, () => {
+    beforeEach(() => {
+        cy.login();
+
+        Navigation.visit("dashboard/drag-drop-widgets");
+        new EditMode().isInEditMode();
+        new InsightsCatalog().waitForCatalogReload();
+    });
+
+    it("shows placeholder text during drag", () => {
+        //create a new row, between 2 existing rows
+        dashboard.getRow(1).dragAbove("ComboChart");
+        dashboard.hasPlaceholderText("Drop to create a new section");
```

Comment:
hasPlaceholderText() is defined in dashboard because the placeHolder text is on dashboard. In line 35, it's calling to getRow(), it's a layout row. During in line 40, it's also use the hasPlaceholderText(), but the line 40 is calling to getWidget(). We can't merge them